e2e/config/config.go

```diff
@@ -15,7 +15,7 @@ var (
 	Azure             = mustNewAzureClient()
 	ResourceGroupName = "abe2e-" + Config.Location
```

Comment:
We will want to revert these changes when merging the PR. Then once this PR is merged, delete our e2e resource group and re-run an airgap test and the resources will get recreated.